docs/tutorial/twitter/tutorial-part-3.md

```diff
@@ -184,10 +204,9 @@ Now it should be the way we like it to be. Sure it's not perfect because it has
 Posting tweets
 --------------
 
-As you have seen in the last paragraphs, creating the data access layer is not that hard using qooxdoo's data binding. That is why we want you to implement the rest of the application: Posting of tweets. But we will give you some hints so it does not take that much time for you.
+As you have seen in the last paragraphs, creating the data access layer is not that hard using qooxdoo's data binding. That is why we want you to implement the rest of the application: Posting of tweets. 
```

Comment:
replace paragraph with:
As you have seen previously, creating the data access layer is easy with qooxdoo's data binding. We believe, using what you learned of data binding, that you now have what you need to implement the remainder of the application: posting of tweets.